README.md

```diff
@@ -219,11 +219,14 @@ options:
   --auto-deps-management
                         Automatically manages dependencies that are not covered by ROS ESM.
                         Their source code is retrieved and compiled against ROS ESM.
+  --install-debs        Install the released packages so they are available.
```

Comment:
I'd suggest `--install-released-packages` so that it is clearer.

```suggestion
  --install-released-packages        Progressively install released packages so that dependencies are satisfied.
```